app/templates/common-address-in-scotland.html

```diff
@@ -10,7 +10,11 @@
 
 {% block main %}
 
-    <h1 class="u-mt-l">{{ _('This address is not part of the census for England and Wales') }}</h1>
+    {% if display_region == 'ni' %}
+        <h1 class="u-mt-l">This address is not part of the census for Northern Ireland</h1>
```

Comment:
Those are mark-ups for the babel translation - anything with those mark-ups are picked up and listed for translating in the messages files. In this case, because the ni version is only ever in english, it doesn't need marking up

Reply:
That's interesting, thanks.